### PR TITLE
SEO-2190_Site_Audit_Issues

### PR DESCRIPTION
**Pull request by gowthamanthavasiyappan**
Hi @Aishwarya-Ganesan,

Worked on site audit issues, merge these changes.

Regards,
Gowthaman

### CODE REVIEW

File: WindowsForms/DataGrid/Rows.md [hunk 1]
```diff
@@ -70,15 +70,15 @@ Me.sfDataGrid1.Style.RowHeaderStyle.SelectionBackColor = Color.White
 </tr>
 <tr>
 <td>
-<img src= "Rows_images/AddNewRowIcon.png"/>
+<img src= "Rows_images/AddNewRowIcon.png" alt="WinForms DataGrid AddNewRowIcon"/>
```
Comment (Aishwarya-Ganesan) [inline WindowsForms/DataGrid/Rows.md:73]: why it is added like html tag? is this correct?

Comment (Aishwarya-Ganesan) [inline WindowsForms/DataGrid/Rows.md:73]: why it is added like html tag? is this correct?

Comment (gowthamanthavasiyappan) [reply WindowsForms/DataGrid/Rows.md:73]: Yes, images are placed in html tag, that's why I added alt text in that format.

**Review by Aishwarya-Ganesan — APPROVED** [review]
Changes are fine

**Pull request merged**